### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by oysteoh**

### CODE REVIEW

**Comment by ManInFez** [comment]
Maybe we could create a `response-model`, which would contain all the data which we would need, and the axis information would be in `response.axis`? The data_loader would then create a response model?

**Comment by oysteoh** [comment]
> Maybe we could create a `response-model`, which would contain all the data which we would need, and the axis information would be in `response.axis`? The data_loader would then create a response model?

Done! I hope... ? 👍  If this is what you meant? Currently it's looking a bit weird since the controller pulls out realizations and observations as well from the response - but i will take this bit by bit while discussing with you !

File: ertviz/controllers/time_series_controller.py [hunk 1]
```diff
@@ -113,13 +107,14 @@ def _update_graph(value):
...
+
+        x_axis = response.axis
+        realizations = _get_realizations_data(response.realizations, x_axis)
         observations = []
 
-        for obs in response.get("observations", []):
-            observations += _get_observation_data(obs["data"], x_axis)
+        for obs in response.observations:
+            observations += _get_observation_data(obs, x_axis)
         print(len(observations))
```
Comment (ManInFez) [inline ertviz/controllers/time_series_controller.py:118]: remove print

**Review by ManInFez — APPROVED** [review]
minor comment

**Pull request merged**